packages/core-components/src/components/TopBar.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { ReactNode } from 'react';
+
+export type TopBarProps = {
+  mobileMode?: boolean;
+  className?: string;
+  brandIcon?: ReactNode;
+  phoneIcon?: ReactNode;
+  languageButton?: ReactNode;
+  linkContact1?: string;
+  linkContact2?: string;
+  phoneNumber1?: string;
+  phoneNumber2?: string;
+  nameContact1?: string;
+  nameContact2?: string;
```

Comment:
@Neeze change TopBarProps only contains topBarItem: TopBarItemType

Something like this....
```suggestion
  export const TopBarItemType = {
    phoneNumber: string;
    name: string;
  }

  topBarItems[]: TopBarItemType[];
```
We do not need brandIcon by now, we need to confirm the mobile layout before implementing